server/s3_object_metadata.cc

```diff
@@ -805,23 +805,7 @@ std::string S3ObjectMetadata::to_json() {
 // Streaming to json
 std::string S3ObjectMetadata::version_entry_to_json() {
```

Comment:
Also version_key_null_in_index is applicable for to_json only, we need to handling this while versioning is suspended

Reply:
@sjain09 @keithpine Lets keep this version_entry_to_json() API as well. Currently we will call the to_json() directly but in future it might help with handling the differences between the two metadata.

Reply:
so, for time being we are keeping it.